decoder/semantic_tokens.go

```diff
@@ -164,42 +172,29 @@ func (d *PathDecoder) tokensForExpression(ctx context.Context, expr hclsyntax.Ex
 		if err != nil {
 			return tokens
 		}
+
 		countAvailable := schema.ActiveCountFromContext(ctx)
 		countIndexAttr := lang.Address{
-			lang.RootStep{
-				Name: "count",
-			},
-			lang.AttrStep{
-				Name: "index",
-			},
+			lang.RootStep{Name: "count"}, lang.AttrStep{Name: "index"},
```

Comment:
No problem. In adding each.*, I wondered if these addresses should be extracted out of this function, or if this code should be a switch statement instead of an if/else since we expect to add more (dynamic, etc).

Reply:
It's something I plan to address as part of go-to-definition/go-to-references - I'm hoping that once we treat `each.*` as actual reference {target,origin} then we can just leverage the existing reference matching logic and this address here shouldn't be necessary as we compare it against _all_ valid references instead.

TL;DR I wouldn't worry about it here.